### ISSUE TITLE
Objective C -> Swift

### ISSUE THREAD

**Issue by eddielement** [suicidesafetyplan/safetyplan-ios#6]
Translate the code from Objective C to Swift to be more futureproof.

**Comment by shayneptorres** [comment]
I would love to start work on this issue. I think it best that we focus on individual view controllers and create branches with names the reflect the view controller being converted. I also mentioned in [this issue](https://github.com/suicidesafetyplan/safetyplan-ios/issues/10) that we should establish a gitflow and that all work should be done off a develop branch so everyone isnt merging everything into `master`

Unless consulted to do otherwise. I will begin working on the `CrisisViewController`

**Comment by eddielement** [comment]
I like that approach - I bet for a lot of the view controllers it's going to be very straightforward to feel free to combine them into a single branch. For example, all of the Guide view controllers.

**Comment by NinjaSnail42** [comment]
I can work on working on the Guide View Controllers since I am having issues with Dark Mode right now.

**Comment by tomaszpieczykolan** [comment]
Hey guys, I'd like to contribute a little bit as well :) With the new Swift code, I think it's best to agree on some kind of architecture, to keep things consistent.

**Comment by shayneptorres** [comment]
@tomaszpieczykolan I agree. Establishing this early would help limit major refactoring later on. Did you have any ideas? We could stick with the a simple MVC or something like MVVM.

**Comment by eddielement** [comment]
I would lean slightly towards MVC :)

**Comment by shayneptorres** [comment]
I will begin working on all the Safety Plan View Controllers. The Main SafetyVC and all its sub VCs

**Comment by igroti** [comment]
Might be helpful (has helped me in the past)
https://swiftify.com/converter/code/
